src/Maker/MakeForm.php

```diff
@@ -41,6 +51,25 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
             ->addArgument('name', InputArgument::OPTIONAL, sprintf('The name of the form class (e.g. <fg=yellow>%sType</>)', Str::asClassName(Str::getRandomTerm())))
             ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeForm.txt'))
         ;
+
+        $inputConf->setArgumentAsNonInteractive('name');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        if (null === $input->getArgument('name')) {
+            $argument = $command->getDefinition()->getArgument('name');
+            $question = new Question($argument->getDescription());
+            $value = $io->askQuestion($question);
+
+            $input->setArgument('name', $value);
+
+            if ($this->entityHelper->isDoctrineInstalled()) {
```

Comment:
We need to ask the below question in ALL cases, because the user may not use Doctrine, but may want to bind their form to a model class. In that case, of course, the `$this->entityHelper->getEntitiesForAutocomplete()` should return an empty array.